src/Text/Layout/Table/Cell/Formatted.hs

```diff
@@ -11,6 +11,7 @@ module Text.Layout.Table.Cell.Formatted
     ( Formatted
     , plain
     , formatted
+    , mergeAlign
```

Comment:
Now that we export it, can we think of a more descriptive name?

Reply:
I went with alignFromState